crates/driver/src/infra/api/error.rs

```diff
@@ -33,7 +33,10 @@ impl From<Kind> for (hyper::StatusCode, axum::Json<Error>) {
         let description = match value {
             Kind::QuotingFailed => "No valid quote found",
             Kind::SolverFailed => "Solver engine returned an invalid response",
-            Kind::SolutionNotFound => "No solution found for the auction",
+            Kind::SolutionNotAvailable => {
+                "no solution is available yet, this might mean that /settle was called before \
+                 /solve returned"
```

Comment:
How can this happen? (settle being called before solve returned)?

Reply:
Added a `driver` test to document this.

Reply:
9daec2d838cba910f1a737b0e0ff51b146c57188